docs/devguide/modules-dev-guide.asciidoc

```diff
@@ -306,8 +308,19 @@ them, you can build them visually in Kibana and then run the following command:
 
 [source,shell]
 ----
+$ cd dev-tools/cmd/dashboards
+$ make # if export_dashboard is not built
+$ ./export_dashboards -dashboard '{dashboard-id}' -output '../../../filebeat/module/{module}/_meta/kibana/default/dashboard'
+----
+
+New Filebeat modules might not be compatible with Kibana 5.x. But if compatibility is required,
```

Comment:
I'd change the wording to say:

The exported dashboards might not be compatible with Kibana 5.x. To export dashboards that are compatible with 5.x, run the following command inside the developer virtualenv: